### ISSUE TITLE
Expand timezone abbreviations

### ISSUE THREAD

**Issue by mattobee** [mattobee/eventua11y#57]
Abbreviated timezones (e.g. GMT, EST, CET) are displayed next to datetimes. Those abbreviations should be expanded.
